### PR TITLE
#163787644 Implement pep8 standards

### PR DESCRIPTION
**Pull request by asirvex**
**What does this PR do**

Implement linting standards using tox, flake8 and pylint

**Description of Task to be completed?**

Makes sure that the code meets the quality standards

**How should this be manually tested?**

$ on the commandline, run `tox`

####Related PT story
[#163787644](https://www.pivotaltracker.com/story/show/163787644)

### CODE REVIEW

File: .pylintrc [hunk 1]
```diff
@@ -0,0 +1,561 @@
+[MASTER]
```
Comment (sirghiny) [inline .pylintrc:1]: Customise this file to our project. It should only have the paths and errors to ignore.

File: authors/settings.py [hunk 2]
```diff
@@ -96,16 +96,16 @@
 
 AUTH_PASSWORD_VALIDATORS = [
     {
-        'NAME': 'django.contrib.auth.password_validation.UserAttributeSimilarityValidator',
+        'NAME': 'django.contrib.auth.password_validation.UserAttributeSimilarityValidator',     # noqa E501
```
Comment (sirghiny) [inline authors/settings.py]: To ignore an error that appears severally in one file, you can have a comment at the top e.g `#pylint:disable=invalid-name`

File: .pylintrc [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+extension-pkg-whitelist= 
+
+ignore=CVS,migrations
+
+disable=print-statement,
```
Comment (sirghiny) [inline .pylintrc]: Do we really need all these disabled? Just pick the specific ones that we can afford to ignore.

**Comment by PaulinhoNdegwa** [comment]
@asirvex The implementation of the task is great. You have a good understanding of `pylint` and `tox` tools.

**Pull request merged**